src/controllers/user.controller.js

```diff
@@ -27,6 +36,44 @@ export class UserController {
     }
   }
 
+  static async updateUserVerified(req, res){
+	try {
+		const user = req.user;
+		await UserService.updateUser({ verified: true }, user.id);
+		return res.status(201).json({
+			message: 'Welcome! Verified successfully', 
+			user: user,
+		});
```

Comment:
- Status code is a success, not created
- Change message to: "Your account has verified successfully!"